### PR TITLE
List-view

### LINKED ISSUE #2: Make the view for RecyclerView

### PR DESCRIPTION
**Pull request by ris3456**
Fixes: #2

### CODE REVIEW

File: app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        TextView nameTextField=(TextView)rowView.findViewById(R.id.nameTextViewID);
+        TextView infoTextField=(TextView)rowView.findViewById(R.id.infoTextViewID);
+
+        nameTextField.setText(nameArray[position]);
+        infoTextField.setText(infoArray[position]);
+
+        return rowView;
+    }
+}
```
Comment (abhinavraj23) [inline app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java]: @ris3456 Please refactor your code, [StackOverFlowLink](https://stackoverflow.com/questions/16580171/code-formatting-shortcut-in-android-studio)

File: app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        CustomListAdapter cs = new CustomListAdapter(this, nameArray, infoArray);
+        listView = (ListView) findViewById(R.id.listviewID);
+        listView.setAdapter(cs);
+
```
Comment (abhinavraj23) [inline app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java]: Why this extra line, remove it!

File: app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java (hunk 1 above)
Comment (abhinavraj23) [reply app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java]: And @ris3456 why have you made this adapter, this wasn't your task!!!

File: app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java (hunk 2 above)
Comment (abhinavraj23) [reply app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java]: You don't have to make the adapter, your task was just to make a simple xml view

File: app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
     }
+
```
Comment (abhinavraj23) [inline app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java]: Extra line, why???

File: app/src/main/res/layout/listview_row.xml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <LinearLayout
+        android:id="@+id/linearlayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        tools:ignore="MissingConstraints"
+        tools:layout_editor_absoluteX="8dp"
+        tools:layout_editor_absoluteY="8dp">
+
```
Comment (abhinavraj23) [inline app/src/main/res/layout/listview_row.xml]: Ideally, I just wanted this file in the PR, please fix the PR

**Review by abhinavraj23 — CHANGES_REQUESTED** [review]

**Comment by abhinavraj23** [comment]
@ris3456 Please just make the view and not the adapters and all, that wasn't your task!!! Ask me doubts before doing things!

File: app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java (hunk 1 above)
Comment (ris3456) [reply app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java]: @abhinavraj23 srry i was confused

File: app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java [hunk 5]
```diff
@@ -1,13 +1,33 @@
...
 public class MainActivity extends AppCompatActivity {
 
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
```
Comment (abhinavraj23) [inline app/src/main/java/com/developer/abhinavraj/moviesearch/MainActivity.java]: Not required

**Review by abhinavraj23 — CHANGES_REQUESTED** [review]
Please make changes

**Pull request closed without merging**